lib/generators/boring/database_cleaner/active_record/rspec/install/install_generator.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module Boring
+  module DatabaseCleaner
+    module ActiveRecord
+      module Rspec
+        class InstallGenerator < Rails::Generators::Base
+          desc 'Adds database_cleaner-active_record gem to the application for cleaning databases during tests'
+          source_root File.expand_path("templates", __dir__)
+
+          TEST_FRAMEWORK_GEM = 'rspec-rails'
+
+          class_option :skip_js_configs, type: :boolean,
+                                               aliases: "-sj",
+                                               desc: "Skip JacaScript dependent configurations"
+
+          def verify_presence_of_rspec_gem
+            gem_file_content_array = File.readlines("Gemfile")
+            rspec_is_installed = gem_file_content_array.any? { |line| line.include?(TEST_FRAMEWORK_GEM) }
+
+            return if rspec_is_installed
+
+            say "We couldn't find #{TEST_FRAMEWORK_GEM} gem. Please configure #{TEST_FRAMEWORK_GEM} and run the generator again!", :red
+
+            abort
+          end
+
+          def verify_presence_of_rails_helper
+            return if File.exist?("spec/rails_helper.rb")
+
+            say "We couldn't find spec/rails_helper.rb. Please configure RSpec and run the generator again!", :red
+
+            abort
+          end
+
+          def add_database_cleaner_active_record_gem
+            say 'Adding database_cleaner-active_record gem', :green
+
+            Bundler.with_unbundled_env do
+              run "bundle add database_cleaner-active_record --group='test'"
```

Comment:
using `bundle add --group` adds new line to the gem file and I feel most people(including me) would be annoyed if all test group gems are not clubbed under one block.

I would recommend using `insert_into_file` with regex into the test group. We have already done that in some of the generator so should be easy to copy here. 

WDYT?